### ISSUE TITLE
Move `sendTargetingParams` To Commercial Service

### ISSUE THREAD

**Issue by JamieB-gu** [guardian/dotcom-rendering#12243]
Request from Android.

https://github.com/guardian/bridget/blob/a36a12c89bdba7d1d936d9d9523f0fa83f0c1fee/thrift/native.thrift#L170-L172

to

https://github.com/guardian/bridget/blob/a36a12c89bdba7d1d936d9d9523f0fa83f0c1fee/thrift/native.thrift#L104-L107

Keep both for a while for older apps versions.

Will need updates in both iOS and Android.

**Comment by CDicksonG** [comment]
We need to speak to the Apps Team and check if this is needed

**Comment by EnochHankombo92** [comment]
> We need to speak to the Apps Team and check if this is needed @EnochHankombo92, is this something that we should be doing?

Yes this was a request from us on Android, which will save us a lot of pain going forward. Very much needed.

**Comment by arelra** [comment]
@EnochHankombo92 is the requirement just to move the service that the method is under - from Analytics to Commercial? The method itself wouldn't change?

**Comment by EnochHankombo92** [comment]
> @EnochHankombo92 is the requirement just to move the service that the method is under - from Analytics to Commercial? The method itself wouldn't change?

That is correct. The method doesn't require any changes, we just need it accessible in Commercial service

**Comment by arelra** [comment]
PR here: https://github.com/guardian/bridget/pull/184

**Comment by arelra** [comment]
Related:
https://theguardian.atlassian.net/browse/LIVE-6938

Tasks for apps team:
- compare the targeting parameters form the web and from mapi to see if we continue using MAPI exclusively for targeting parameters
- ensure its being applied to DCAR articles
- catch up with Android to hear their opinionn
- remove the Bridget function if necessary

**Comment by SiAdcock** [comment]
In progress!
